index.d.ts

```diff
@@ -133,6 +133,17 @@ declare namespace ora {
 		readonly prefixText?: string | PrefixTextGenerator;
 	}
 
+	interface PromiseOptions<T> extends Options {
+		/** The new text of the spinner when the promise is resolved.
+     *
+     * * If undefined, will keep the initial text. */
+		successText?: ((resp: T) => string) | string;
+		/** The new text of the spinner when the promise is rejected.
+     *
+     * If undefined, will keep the initial text. */
```

Comment:
This needs to be improved. Incorrect indentation and the formatting looks messy.

Reply:
Hmmm, I haven't looked at the result. After all the xo's errors, I just `xo --fix` and believed it would fix everything properly. I will soon do your requests.